src/prismic-react/index.js

```diff
@@ -38,6 +38,10 @@ function UniversalComponent({request: fnFetchData, component: WrappedComponent})
       .catch((e) => console.log(e.message));
     }
 
+    componentDidMount() {
```

Comment:
you shouldn't do this, it's done by the script prismic.js which is imported in index.ejs